configs/rpt/usbradio.conf

```diff
@@ -176,6 +176,11 @@ clipledgpio = 0             ; Enable ADC Clip Detect feature to use a GPIO outpu
                             ; Supports URI Clip LED by setting a GPIO (if available) high for 500mS
                             ; when clipping detected. Value = GPIO# to use (GPIO1 recommended)
 
+legacyaudioscaling = no     ; If yes, continue to do raw audio sample scaling and clipping, resulting in Tx audio levels increasing
+                            ; by 0.83dB and Rx audio levels decreasing by 1.94dB. This should be set to no unless you have an
+                            ; existing node with precisely adjusted audio levels but are not yet able to adjust them. If set to
+                            ; yes, degraded audio quality will result
```

Comment:
audio quality will --> audio quality may

Reply:
Do we have to say "degraded audio quality will result"?  My system is setup properly using the original code and I don't have degraded quality.

Reply:
Existing installs with any non-original-CM108 URI, will have degraded audio quality to some extent, though I can see that the term can imply a major perceptible difference which should usually not be the case. There is scaling and clipping happening in both rx and tx audio which may result in dynamic range being reduced by a couple dB or average THD increased by ~1%, which would not be perceptible in most systems. But the important thing is just to get it fixed so I'll simplify some of the text a bit and try to avoid any subjective terms.